client/components/post-card/compact.jsx

```diff
@@ -15,43 +15,43 @@ import safeImageUrl from 'lib/safe-image-url';
 import resizeImageUrl from 'lib/resize-image-url';
 import AuthorAndSite from './author-and-site';
 
-export function SmallPostCard( { translate, post, site, onPostClick = noop, onSiteClick = noop } ) {
-	const classes = classnames( 'post-card small', {
+export function CompactPostCard( { translate, post, site, onPostClick = noop, onSiteClick = noop } ) {
```

Comment:
the compact variant here has a different DOM than the non-compact one. Maybe `Small` was a better name?